packages/core/client/src/application/schema-initializer/components/SchemaInitializerItem.tsx

```diff
@@ -73,9 +73,9 @@ export const SchemaInitializerItem = memo(
           children: childrenItems,
         },
       ];
-    }, [name, style, className, children, title, onClick, icon, childrenItems]);
+    }, [name, disabled, style, className, children, title, onClick, icon, childrenItems]);
 
-    if (items && items.length > 0) {
+    if (items && items.length > 0 && !disabled) {
       return <SchemaInitializerMenu items={menuItems}></SchemaInitializerMenu>;
```

Comment:
```suggestion
      return <SchemaInitializerMenu items={menuItems} disabled={disabled}></SchemaInitializerMenu>;
```
SchemaInitializerMenu 组件好像支持传一个 disabled 属性。